<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>640</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>ENKA-S</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../resources.qrc">
    <normaloff>:/MainWindow/img/icon.png</normaloff>:/MainWindow/img/icon.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Open Sans</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="locale">
       <locale language="C" country="AnyTerritory"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabHome">
       <attribute name="title">
        <string>Home</string>
       </attribute>
       <widget class="SystemPreview" name="oglHomeScreen">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>981</width>
          <height>591</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="locale">
         <locale language="C" country="AnyTerritory"/>
        </property>
       </widget>
       <widget class="QWidget" name="widget" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>540</y>
          <width>971</width>
          <height>44</height>
         </rect>
        </property>
        <property name="locale">
         <locale language="C" country="AnyTerritory"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="locale">
            <locale language="C" country="AnyTerritory"/>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>538</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_45">
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="locale">
            <locale language="C" country="AnyTerritory"/>
           </property>
           <property name="text">
            <string>v2.0.0</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>981</width>
          <height>110</height>
         </rect>
        </property>
        <property name="locale">
         <locale language="C" country="AnyTerritory"/>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_43">
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>50</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="locale">
            <locale language="C" country="AnyTerritory"/>
           </property>
           <property name="text">
            <string>ENKA-S</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_46">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="font">
            <font>
             <family>Open Sans</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="locale">
            <locale language="C" country="AnyTerritory"/>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="NewSimulationTab" name="tabNewSimulation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>New Simulation</string>
       </attribute>
      </widget>
      <widget class="LoadSimulationTab" name="tabLoadSimulation">
       <attribute name="title">
        <string>Load Simulation</string>
       </attribute>
      </widget>
      <widget class="LogsTab" name="tabLogs">
       <attribute name="title">
        <string>Logs</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SystemPreview</class>
   <extends>QOpenGLWidget</extends>
   <header>widgets/system_preview.h</header>
  </customwidget>
  <customwidget>
   <class>LoadSimulationTab</class>
   <extends>QWidget</extends>
   <header>views/load_simulation_tab/load_simulation_tab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>NewSimulationTab</class>
   <extends>QWidget</extends>
   <header>views/new_simulation_tab/new_simulation_tab.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LogsTab</class>
   <extends>QWidget</extends>
   <header>views/logs_tab/logs_tab.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
